src/Language/Haskell/Refact/API.hs

```diff
@@ -78,6 +81,7 @@ module Language.Haskell.Refact.API
 
        , setStateStorage
        , getStateStorage
+       , fetchAnnsFinal
```

Comment:
I am not sure that this is strictly necessary.

1. `putRefactParsed` can take a second argument of `mempty`, and usually does.  There should perhaps be a separate `putRefactParsedWithAnns` for the more rare case.

2. Most annotations updates can be done via `modifyRefactAnns`, which makes it easier to re-arrange code as you do not have to always makes sure you are updating the right annotations variable. It is in the `RefactGhc` state, after all.